Cookbook/Technical-Documents/TechnicalDebt.md

```diff
@@ -7,111 +7,246 @@ Technical debt and legacy code has a sad tendency to build up in any software pr
-| SignInViewModel | AV-332 |
-| NHSRegistrationStep1ViewController | NRX-361 |
-| NHSRegistrationStep2ViewController | NRX-186 |
+| View Model | JIRA tickets | Code Ownership | Comments |
+| ---------- | -------- | -------- | -------- |
+| AppointmentCancellationViewModel | ~CNSMR-950, CNSMR-951~ | Booking/Native Apps | Will be retired once the iOS part of CNSMR-1737 has been completed. |
+| AppointmentConsultationNotesViewModel | ~CNSMR-952, CNSMR-953~ | Booking/Native Apps | Will be retired once the iOS part of CNSMR-1737 has been completed. |
+| AppointmentDetailsViewModel | ~CNSMR-954, CNSMR-955~ | Booking/Native Apps | Will be retired once the iOS part of CNSMR-1737 has been completed. |
+| AppointmentNotesViewModel | ~CNSMR-956, CNSMR-957~ | Booking/Native Apps | Will be retired once the iOS part of CNSMR-1737 has been completed. |
+| AppointmentPrescriptionViewModel | ~CE-30~ | Booking/Native Apps | Will be retired once the iOS part of CNSMR-1737 has been completed. |
+| AppointmentReferralViewModel | ~CNSMR-958, CNSMR-959~ | Booking/Native Apps | Will be retired once the iOS part of CNSMR-1737 has been completed. |
+| AppointmentReferralsViewModel | ~CNSMR-960, CNSMR-961~ | Booking/Native Apps | Will be retired once the iOS part of CNSMR-1737 has been completed. |
+| AppointmentReplayViewModel | ~CNSMR-962, CNSMR-963~ | Booking/Native Apps | Will be retired once the iOS part of CNSMR-1737 has been completed. |
+| AppointmentListViewModel | ~CNSMR-964, CNSMR-965~ | Booking/Native Apps | Used when `isNewAppointmentsEnabled == false`. Will be retired once the iOS part of CNSMR-1737 has been completed. |
+| BookAppointmentViewModel | ~CNSMR-966, CNSMR-967~ | Booking/Native Apps | Will be retired once the iOS part of CNSMR-1737 has been completed. |
+| PractitionerDetailsViewModel | ~CNSMR-968, CNSMR-969~ | Booking/Native Apps | Will be retired once the iOS part of CNSMR-1737 has been completed. |
+| ChatHistoryViewModel | CNSMR-970, CNSMR-971 | | |
+| MedicalHistoryViewModel | CNSMR-972, CNSMR-973 | | |
+| PersonalDetailsViewModel | CNSMR-974, CNSMR-975 | | |
+| InfoItemsViewModel | CNSMR-976, CNSMR-977 | | |
+| GPDetailsViewModel | CNSMR-978, CNSMR-979 | | |
+| AddAddressViewModel | CNSMR-980, CNSMR-981 | | |
+| PrivacySettingsViewModel | AV-334 | | |
+| ChooseCountryViewModel | AV-338 | | |
+| ForgotPasswordViewModel | CNSMR-904 | | Used when `isNewForgotPasswordEnabled == false` |
+| NotificationsViewModel | AV-342 | | Used when `isNewNotificationsEnabled == false` |
+| SignInViewModel | AV-332 | | Used when `isNewSigninEnabled == false`|
+| NHSRegistrationStep1ViewController | NRX-361 | | Used when `isV4NHSRegistrationStep1And2Enabled == false` |
+| NHSRegistrationStep2ViewController | NRX-186 | | Used when `isV4NHSRegistrationStep1And2Enabled == false` |
+| RedemptionViewModel | CNSMR-921 | | Used when `isNewRedemptionEnabled ==false ` |
+| OnboardingViewModel | AV-118 | | Used when `isNewWelcomeScreenEnabled == false` |
```

Comment:
Just to note, I'm working on refactoring `OnboardingViewModel` in [CNSMR-2581](https://babylonpartners.atlassian.net/browse/CNSMR-2581) as my objective

Reply:
Nice. I have updated the table with the ticket number.